.github/workflows/generate-pinval.yaml

```diff
@@ -43,18 +44,31 @@ env:
   AWS_ATHENA_S3_STAGING_DIR: s3://ccao-athena-results-us-east-1/
   HUGO_VERSION: "0.147.5"
 
+defaults:
+  run:
+    # Always use builtin bash shell to take advantage of automatic -eo pipefail.
+    # See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrunshell
+    shell: bash
```

Comment:
This config allows us to remove the manual `set -eo pipefail` calls from the bash scripts below.